specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/Diagnostics.json

```diff
@@ -15,6 +15,130 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/detectors": {
```

Comment:
You are adding new api, and in public repo, it requires to be in a new api-version.